### PR TITLE
Update zh-TW translations 

### PR DESCRIPTION
**Pull request by KuwaLee**
@pfrazee I'm so sorry that my wrong operations of merge conflict cause https://github.com/bluesky-social/social-app/pull/3960 being closed.

### CODE REVIEW

**Review by auroursa — COMMENTED** [review]
LGTM

**Comment by KuwaLee** [comment]
@pfrazee This PR is ready to go!

File: src/locale/locales/zh-TW/messages.po [hunk 1]
```diff
@@ -2624,14 +2569,18 @@ msgstr "已解除靜音的列表"
 msgid "Lists"
 msgstr "列表"
 
+#: src/components/dms/ConvoMenu.tsx:255
+msgid "Lists blocking this user:"
+msgstr "列表已封鎖這個用戶："
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2574]:
```suggestion
msgstr "封鎖此用戶的列表："
```

File: src/locale/locales/zh-TW/messages.po [hunk 2]
```diff
@@ -3220,11 +3162,11 @@ msgstr "至少有一張圖片缺失了替代文字。"
 
 #: src/screens/Onboarding/StepProfile/index.tsx:120
 msgid "Only .jpg and .png files are supported"
-msgstr ""
+msgstr "僅支援 .jpg 和 .png 文件"
 
 #: src/view/com/threadgate/WhoCanReply.tsx:100
 msgid "Only {0} can reply."
-msgstr "只有 {0} 可以回覆。"
+msgstr "只有{0}可以回覆。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:3169]:
```suggestion
msgstr "只有 {0} 可以回覆。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 3]
```diff
@@ -3246,7 +3188,7 @@ msgstr "開啟"
 
 #: src/screens/Onboarding/StepProfile/index.tsx:280
 msgid "Open avatar creator"
-msgstr ""
+msgstr "開啟頭像創建器"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:3191]:
```suggestion
msgstr "開啟頭像創建工具"
```

File: src/locale/locales/zh-TW/messages.po [hunk 4]
```diff
@@ -4947,7 +4859,7 @@ msgstr "版權政策已移動到 <0/>"
 
 #: src/view/com/posts/FeedShutdownMsg.tsx:66
 msgid "The feed has been replaced with Discover."
-msgstr ""
+msgstr "動態源已由「Discover」取代。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:4862]:
```suggestion
msgstr "此動態源已由「Discover」取代。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 5]
```diff
@@ -5438,14 +5351,10 @@ msgstr "取消訂閱"
...
 
-#: src/lib/moderation/useReportOptions.ts:85
-msgid "Unwanted sexual content"
-msgstr ""
-
 #: src/lib/moderation/useReportOptions.ts:71
 #: src/lib/moderation/useReportOptions.ts:84
 msgid "Unwanted Sexual Content"
-msgstr "無關情色的內容"
+msgstr "不受歡迎的情色的內容"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:5357]:
```suggestion
msgstr "不受歡迎的情色內容"
```

File: src/locale/locales/zh-TW/messages.po [hunk 6]
```diff
@@ -5122,25 +5038,25 @@ msgstr "沒有 Bluesky 帳號，無法查看此內容。"
 
 #: src/view/screens/Settings/ExportCarDialog.tsx:94
 msgid "This feature is in beta. You can read more about repository exports in <0>this blogpost</0>."
-msgstr "此功能目前為測試版本。您可以在<0>這篇部落格文章</0>中了解更多有關資訊。"
+msgstr "此功能目前為測試版本。您可以在 <0> 這篇部落格文章 </0> 中了解更多有關資訊。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:5041]:
```suggestion
msgstr "此功能目前為測試版本。您可以在<0>這篇部落格文章</0>中了解更多有關資訊。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 7]
```diff
@@ -5122,25 +5038,25 @@ msgstr "沒有 Bluesky 帳號，無法查看此內容。"
...
+msgstr "這裡是空的！"
 
 #: src/view/com/posts/CustomFeedEmptyState.tsx:37
 msgid "This feed is empty! You may need to follow more users or tune your language settings."
 msgstr "這個動態源是空的！您或許需要先跟隨更多的人或檢查您的語言設定。"
 
 #: src/view/com/posts/FeedShutdownMsg.tsx:97
 msgid "This feed is no longer online. We are showing <0>Discover</0> instead."
-msgstr ""
+msgstr "此動態源已不再上線。取而代之的是<0>「Discover」動態源</0>。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:5059]:
```suggestion
msgstr "此動態源已經下線。我們將展示「<0>Discover</0>」動態源。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 8]
```diff
@@ -2668,15 +2617,15 @@ msgstr "看起來像是 XXXXX-XXXXX"
...
 msgid "Looks like you unpinned all your feeds. But don't worry, you can add some below 😄"
-msgstr ""
+msgstr "看起來您已取消釘選所有動態源，但不用擔心，您可以在下面添加一些😄"
 
-#: src/screens/Feeds/NoFollowingFeed.tsx:38
-msgid "Looks like you're missing a following feed."
-msgstr ""
+#: src/screens/Feeds/NoFollowingFeed.tsx:37
+msgid "Looks like you're missing a following feed. <0>Click here to add one.</0>"
+msgstr "您看起來少了一個「Following」動態源，<0>點擊這裡來添加一個。</0>"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2628]:
```suggestion
msgstr "您看起來需要「Following」動態源，<0>點選這裡來新增。</0>"
```

File: src/locale/locales/zh-TW/messages.po [hunk 9]
```diff
@@ -857,10 +831,6 @@ msgstr "清除所有資料"
 msgid "click here"
 msgstr "點擊這裡"
 
-#: src/screens/Feeds/NoFollowingFeed.tsx:46
-msgid "Click here to add one."
-msgstr ""
-
 #: src/components/TagMenu/index.web.tsx:138
 msgid "Click here to open tag menu for {tag}"
 msgstr "點擊這裡以開啟 {tag} 的標籤選單"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:836]:
```suggestion
msgstr "點選這裡以開啟 {tag} 的標籤選單"
```

File: src/locale/locales/zh-TW/messages.po [hunk 10]
```diff
@@ -857,10 +831,6 @@ msgstr "清除所有資料"
 msgid "click here"
 msgstr "點擊這裡"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:832]:
```suggestion
msgstr "按一下這裡"
```

File: src/locale/locales/zh-TW/messages.po [hunk 11]
```diff
@@ -4972,7 +4884,7 @@ msgstr "隱私政策已移動到 <0/>"
 
 #: src/view/screens/Support.tsx:36
 msgid "The support form has been moved. If you need help, please <0/> or visit {HELP_DESK_URL} to get in touch with us."
-msgstr "支援表單已移至別處。如果需協助，請點擊<0/>或前往 {HELP_DESK_URL} 與我們聯繫。"
+msgstr "支援表單已移至別處。如果需協助，請點擊 <0/> 或前往 {HELP_DESK_URL} 與我們聯繫。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:4887]:
```suggestion
msgstr "支援表單已移至別處。如果需協助，請點選 <0/> 或前往 {HELP_DESK_URL} 與我們聯繫。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 12]
```diff
@@ -1796,7 +1758,7 @@ msgstr "外部媒體"
 #: src/components/dialogs/EmbedConsent.tsx:71
 #: src/view/screens/PreferencesExternalEmbeds.tsx:67
 msgid "External media may allow websites to collect information about you and your device. No information is sent or requested until you press the \"play\" button."
-msgstr "外部媒體可能允許網站收集有關您和您裝置的資料。在您按下「播放」按鈕之前，不會傳送或請求任何資料。"
+msgstr "外部媒體可能允許網站收集有關您和您裝置的資料。在您按下「播放」按鈕之前，將不會傳送或請求任何資料。"
```
[the remark this answers is not in the document]
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:1761]: 這裡我改過，但後來還是覺得"將"比較通順

File: src/locale/locales/zh-TW/messages.po (hunk 8 above)
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:2628]: aawesome! I like it!

File: src/locale/locales/zh-TW/messages.po (hunk 10 above)
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:832]: 感覺這兩個都行，但是考慮到bluesky對於太長的字串容易溢出，所以我傾向原句

File: src/locale/locales/zh-TW/messages.po (hunk 11 above)
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:4887]: 看了原碼 <0/> 插入的就是點擊這裡

File: src/locale/locales/zh-TW/messages.po (hunk 9 above)
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:836]: I prefer "點擊"

**Comment by KuwaLee** [comment]
@cirx1e Thanks for your review, plus, would like to receive notification while I opening PR?

File: src/locale/locales/zh-TW/messages.po (hunk 12 above)
[the remark this answers is not in the document]
Comment (cirx1e) [reply src/locale/locales/zh-TW/messages.po:1761]: 「將」是就會的意思
加在這裡有點多餘

https://dict.concised.moe.edu.tw/dictView.jsp?ID=23427

**Comment by cirx1e** [comment]
> @cirx1e Thanks for your review, plus, would like to receive notification while I opening PR?

Sure!

**Comment by KuwaLee** [comment]
label: all previous review conversations are resolved

**Review by cirx1e — COMMENTED** [review]
LGTM

**Comment by KuwaLee** [comment]
label: merge is ready

**Comment by KuwaLee** [comment]
label: merge is ready

**Comment by KuwaLee** [comment]
label: merge is ready

File: src/locale/locales/zh-TW/messages.po (hunk 2 above)
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po]: This should be with no spaces, lol, i forget and sunmit your suggestion

**Comment by KuwaLee** [comment]
label: merge is ready

**Comment by KuwaLee** [comment]
label: merge is ready

**Comment by KuwaLee** [comment]
label: merge is ready

File: src/locale/locales/zh-TW/messages.po [hunk 13]
```diff
@@ -2092,17 +2113,17 @@ msgstr "前往首頁"
...
 
-#: src/components/dms/ConvoMenu.tsx:166
+#: src/components/dms/ConvoMenu.tsx:148
 msgid "Go to user's profile"
 msgstr "前往用戶個人頁面"
 
 #: src/lib/moderation/useGlobalLabelStrings.ts:46
 msgid "Graphic Media"
-msgstr "影像媒體"
+msgstr "成人影像媒體"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2126]:
```suggestion
msgstr "不適宜媒體"
```

https://github.com/bluesky-social/social-app/blob/f93a81bdcf4421b7172f7220f1b2b796e41623b7/src/locale/locales/zh-TW/messages.po#L1760-L1762

File: src/locale/locales/zh-TW/messages.po [hunk 14]
```diff
@@ -3381,6 +3413,10 @@ msgstr "其他帳號"
 msgid "Other..."
 msgstr "其他…"
 
+#: src/screens/Messages/Conversation/ChatDisabled.tsx:18
+msgid "Our moderators have reviewed reports and decided to disable your access to chats on Bluesky."
+msgstr "我們的內容管理者已審核檢舉，並決定禁止您進入在 Bluesky 上的對話。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:3418]:
```suggestion
msgstr "我們的內容管理者已審核檢舉，並決定停用您在 Bluesky 上的私人訊息功能。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 15]
```diff
@@ -3960,7 +4016,7 @@ msgstr "要求發佈前提供替代文字"
 
 #: src/view/screens/Settings/Email2FAToggle.tsx:51
 msgid "Require email code to log into your account"
-msgstr "需要電子郵件驗證碼才能登入您的帳戶"
+msgstr "需要電子郵件驗證碼才能登入您的帳號"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:4019]:
```suggestion
msgstr "登入您的帳號時要求電子郵件驗證碼"
```

or "登入時要求電子郵件驗證碼"

File: src/locale/locales/zh-TW/messages.po [hunk 16]
```diff
@@ -2347,6 +2368,10 @@ msgstr "輸入您的託管服務供應商"
 msgid "Input your user handle"
 msgstr "輸入您的帳號代碼"
 
+#: src/components/dms/MessagesNUX.tsx:79
+msgid "Introducing Direct Messages"
+msgstr "私人訊息已推出"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2373]:
```suggestion
msgstr "為您隆重介紹「私人訊息」"
```

File: src/locale/locales/zh-TW/messages.po [hunk 17]
```diff
@@ -1322,10 +1330,14 @@ msgstr "生動的替代文字"
...
 msgstr "有什麼想說的嗎？"
 
-#: src/view/screens/Settings/index.tsx:468
+#: src/view/screens/Settings/index.tsx:471
 msgid "Dim"
 msgstr "昏暗"
 
+#: src/components/dms/MessagesNUX.tsx:85
+msgid "Direct messages are here!"
+msgstr "隆重推出私人訊息！"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:1339]:
```suggestion
msgstr "私人訊息已推出！"
```

**Comment by KuwaLee** [comment]
label: it's ready to merge

**Comment by KuwaLee** [comment]
label: it's ready to merge

**Review by pfrazee — APPROVED** [review]
Thank you everybody!!

**Pull request merged**